packages/core/test/financial-templates/common/financial-product-libraries/long-short-pair-libraries/SuccessTokenLongShortPairFinancialProductLibrary.js

```diff
@@ -0,0 +1,141 @@
+const hre = require("hardhat");
+const { runDefaultFixture } = require("@uma/common");
+const { getContract } = hre;
+const { didContractThrow, ZERO_ADDRESS } = require("@uma/common");
+const { assert } = require("chai");
+
+// Tested Contract
+const SuccessTokenLongShortPairFinancialProductLibrary = getContract(
+  "SuccessTokenLongShortPairFinancialProductLibrary"
+);
+
+const LongShortPairMock = getContract("LongShortPairMock");
+
+const { toWei, toBN } = web3.utils;
+const strikePrice = toWei("400");
+const basePercentage1 = toBN(toWei("0.5"));
+const basePercentage2 = toBN(toWei("0.2"));
+const collateralPerPair = toBN(toWei("1"));
+
+describe("SuccessTokenLongShortPairFinancialProductLibrary", function () {
+  let successTokenLSPFPL;
+  let lspMock;
+  let accounts;
+
+  before(async () => {
+    await runDefaultFixture(hre);
+    accounts = await hre.web3.eth.getAccounts();
+  });
+
+  beforeEach(async () => {
+    successTokenLSPFPL = await SuccessTokenLongShortPairFinancialProductLibrary.new().send({ from: accounts[0] });
+    lspMock = await LongShortPairMock.new(
+      "1000000", // _expirationTimestamp
+      collateralPerPair // _collateralPerPair
+    ).send({ from: accounts[0] });
+  });
+  describe("Long Short Pair Parameterization", () => {
+    it("Can set and fetch valid strikes", async () => {
```

Comment:
Should add asserts that you can't set `strikePrice`/`basePercentage` equal to 0. General rule of thumb is that every single require statement should be tested once in a unit test.